crates/dojo-lang/src/plugin_test_data/system

```diff
@@ -137,7 +137,7 @@ mod Move {
         calldata.append('name');
         commands::execute(SpawnSystem, calldata.span());
 
-        move_inner(@positions_query, 0_u32, positions_query.len());
+        // move_inner(@positions_query, 0_u32, positions_query.len());
```

Comment:
@tarrencev I've commented this out because it made the tests fail and I'm not sure what's it about 😬

Reply:
ah ok i can look at that